providers/gce/gce.go

```diff
@@ -283,6 +283,11 @@ func (g *Cloud) ContainerService() *container.Service {
 	return g.containerService
 }
 
+// StackType returns the type of the IP protocol stack.
```

Comment:
ut the more I look into this the less I like it, checking all code surrounding it really means a GCE value not a GKE value :/ 

I  wonder if we just should use a flag and call it a day

Reply:
This is confusing indeed, in fact the `g.stackType` field represents the GKE cluster stack type, not the network (GCE) stack type. Maybe this just needs to be plugged in a different way (not via gce.go)? Is this what you are suggesting?

Reply:
yes, let's do a quick brainstorm tomorrow together, maybe a flag or one of the existing flags, this seems a stretch , despite the meaning is cluster, the place is clearly GCE , or node or subnet, not Pods 

/hold

Reply:
Ok, the decision is to use a different mechanism to find out what is the right cluster stack type - instead of using the stack type property defined in /etc/gce.conf, I'll use the `service-cluster-ip-range` flag, which is initialized to either a single CIDR or two CIDRs, depending on the cluster's stack type. I'll close this pull request and prepare a new one.